dojo/tools/dependency_track/parser.py

```diff
@@ -144,7 +144,7 @@ def _convert_dependency_track_finding_to_dojo_finding(self, dependency_track_fin
         # We should collect all the vulnerability ids, the FPF format can add additional IDs as aliases
         # we add these aliases in the vulnerability_id list making sure duplicate findings get correctly deduplicated
         # older version of Dependency-track might not include these field therefore lets check first
-        if 'aliases' in dependency_track_finding['vulnerability']:
+        if 'aliases' in dependency_track_finding['vulnerability'] and len(dependency_track_finding['vulnerability']['aliases']) != 0:
```

Comment:
There is a better format for this check:
```suggestion
        if dependency_track_finding['vulnerability'].get('aliases'):
```
It check both null or empty.